_posts/en/2016-02-02-create-range-0...n-easily-using-one-line.md

```diff
@@ -27,22 +30,30 @@ Array.apply(null, {length: N}).map(Function.call, Number);
 For a more thorough explanation, go [here](https://github.com/gromgit/jstips-xe/blob/master/tips/33.md).
 
 ### Solution 2 (requires ES6)
+
 ```js
 Array.from(new Array(N),(val,index)=>index);
 ```
+
 #### Brief explanation
+
 1. `A = new Array(N)` returns an array with `N` _holes_ (i.e. `A = [,,,...]`, but `A[x] = undefined` for `x` in `0...N-1`).
 2. `F = (val,index)=>index` is simply `function F (val, index) { return index; }`
-3. `Array.from(A, F)` returns an `N`-element array, whose index `I` gets the results of `F(A[I], I) == I`.
+3. `Array.from(A, F)` returns an `N`-element array, whose index `I` gets the results of `F(A[I], I) => I`.
```

Comment:
In hindsight, `==` is probably confusing, but `=>` suggests "fat arrow" syntax, which is also confusing.

Perhaps `F(A[I], I) --> I` would be best.